.cicd/test-package.docker.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euo pipefail
+
+. "${0%/*}/libfunctions.sh"
+
+echo '--- :docker: Pretest Setup'
+
+perform "docker pull $IMAGE"
+perform "docker run --rm -v \"$(pwd):/eos\" -w '/eos' -it $IMAGE ./.cicd/test-package.run.sh"
```

Comment:
I would escape the dollar sign (`$`) before the `pwd` command in the volume mount flag here. The present command will print something like this before running it...
```bash
docker run --rm -v "/home/cicd/work/eos:/eos" -w '/eos' -it docker.io/eosio/eosio:ubuntu-20.04-master ./.cicd/test-package.run.sh
```
...which is accurate, but not terribly useful to the customer because they will have to edit the command if they are copying and pasting it into their shell. If we escape that dollar sign, it will print something like this:
```bash
docker run --rm -v "$(pwd):/eos" -w '/eos' -it docker.io/eosio/eosio:ubuntu-20.04-master ./.cicd/test-package.run.sh
```
Then people can just copy-pasta.

Reply:
I'm ok with it printing the literal path. It might be slightly verbose. I don't see this being "broken" and it seems like to much work and too many builds to change this.